source/SIL.AppBuilder.Portal.Frontend/src/ui/components/sidebar/display.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
```

Comment:
this looks like it returned from the dead? I don't think we need this file?

Reply:
@giancorzo

Reply:
I'm going to merge master to this branch that would fix it